go.mod

```diff
@@ -21,7 +21,7 @@ require (
 	github.com/hashicorp/go-retryablehttp v0.7.0
 	github.com/hashicorp/golang-lru v0.5.4
 	github.com/jongio/azidext/go/azidext v0.4.0
-	github.com/onsi/ginkgo/v2 v2.6.0
+	github.com/onsi/ginkgo/v2 v2.7.0
```

Comment:
/hold 

@mboersma don't we also need an update to the makefile for this one?

Reply:
Yes indeed, thanks for paying attention. :-)